src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -1001,8 +1001,8 @@ public ApiApp updateApiApp(ApiApp app) throws HelloSignException {
         if (!app.hasClientId()) {
             throw new HelloSignException("Cannot update an ApiApp without a client ID. Create one first!");
         }
-        String url = API_APP_URI + "/" + app.getClientId();
-        return new ApiApp(httpClient.withAuth(auth).withPostFields(app.getPostFields()).put(url).asJson());
+        String url = BASE_URI + API_APP_URI + "/" + app.getClientId();
```

Comment:
@radhack I notice that other ApiApp methods are missing the BASE_URI, can you add there as well?

Reply:
@cmpaul done sir!